wsl-module/README.md

```diff
@@ -0,0 +1,14 @@
+Wakurtosis Load Simualtor (WSL)
+===============================
+
+Kurtosis: https://docs.kurtosis.com/
+
+### How to use:
+
+To build docker image:
+    `sh ./build.sh`
+
+Name of the image is wsl:0.0.1
+
+#### Before using this repository make sure that: 
+For the build step make sure that the Wakurtosis enclave exists and make sure the enclave is running before running the container
```

Comment:
Maybe this is for the previous version? I don't see why the enclave should exists since you are running it as a service inside Kurtosis.

Maybe here we could explain `wsl.yml`, but it is already pretty self explanatory so your choice...

Reply:
Correct. Updated the readme file